src/quantum/azext_quantum/_help.py

```diff
@@ -13,36 +13,47 @@
 
 helps['quantum execute'] = """
     type: command
-    short-summary: Submit a job to run on Azure Quantum, and waits for the result.
+    short-summary: Submit a job to run on Azure Quantum, and wait for the result. Equivalent to `az quantum run`.
```

Comment:
This is less of comment for your PR, but do have context on why we have both if they do the same thing?

Reply:
I never heard what the history was: Maybe Andres would know what the rationale was for this duplication, since those two commands have been around since the beginning.